Fix the operator policy documentation (#6371)
open-cluster-management/rhacm-docs: governance/install_operator.adoc

= Installing an operator by using the _OperatorPolicy_ resource (Technology Preview)

To install Operator Lifecycle Manager (OLM) managed operators on your managed clusters, use an `OperatorPolicy` policy template in a `Policy` definition.

[#create-operator-policy]
== Creating an _OperatorPolicy_ resource to install Quay

See the following operator policy sample that installs the latest Quay operator in the `stable-3.11` channel using the Red Hat operator catalog:

[source,yaml]
----
apiVersion: policy.open-cluster-management.io/v1
kind: Policy
metadata:
  name: install-quay
  namespace: open-cluster-management-global-set
spec:
  disabled: false
  policy-templates:
    - objectDefinition:
        apiVersion: policy.open-cluster-management.io/v1beta1
        kind: OperatorPolicy
        metadata:
          name: install-quay
        spec:
          remediationAction: enforce
          severity: critical
          complianceType: musthave
          subscription:
            channel: stable-3.11
            installPlanApproval: Automatic
            name: quay-operator
            source: redhat-operators
            sourceNamespace: openshift-marketplace
----

> After you add the `OperatorPolicy` policy template, the operatorGroup` and `subscription` objects are created on the cluster by using the controller. This prompts the OLM to complete the rest of the installation process. You can view the health of owned resources in the `.status.Conditions` and `.status.relatedObjects` fields of the `OperatorPolicy` resource on the managed cluster.

Author: ```suggestion
After you add the `OperatorPolicy` policy template, the `operatorGroup` and `subscription` objects are created on the cluster by using the controller. This prompts the OLM to complete the rest of the installation process. You can view the health of owned resources in the `.status.Conditions` and `.status.relatedObjects` fields of the `OperatorPolicy` resource on the managed cluster.
```

> After you add the `OperatorPolicy` policy template, the `operatorGroup` and `subscription` objects are created on the cluster by using the controller. This prompts the OLM to complete the rest of the installation process. You can view the health of owned resources in the `.status.Conditions` and `.status.relatedObjects` fields of the `OperatorPolicy` resource on the managed cluster.

Author: ```suggestion
After you add the `OperatorPolicy` policy template, the `operatorGroup` and `subscription` objects are created on the cluster by using the controller. This prompts OLM to complete the rest of the installation process. You can view the health of owned resources in the `.status.Conditions` and `.status.relatedObjects` fields of the `OperatorPolicy` resource on the managed cluster.
```

Reviewer: Updating to remove anthropomorphism. https://www.ibm.com/docs/en/ibm-style?topic=grammar-anthropomorphism

Does OLM complete the installation automatically or is the user prompted from the OLM tool to complete the installation? 
```suggestion
After you add the `OperatorPolicy` policy template, the `operatorGroup` and `subscription` objects are created on the cluster by using the controller. As a result, the rest of the installation is completed by OLM. You can view the health of owned resources in the `.status.Conditions` and `.status.relatedObjects` fields of the `OperatorPolicy` resource on the managed cluster.
```

Author: Could you please change `complete` to `completed`?

OLM automatically handles the installation.

> After the Governance Policy Framework applies the `OperatorPolicy` policy template, the controller creates the `operatorGroup` and `subscription` objects on the cluster. This prompts OLM to handle the rest of the installation process. The controller reports the health of owned resources in the `.status.Conditions` and `.status.relatedObjects` fields of the `OperatorPolicy` resource on the managed cluster.

Reviewer: Removing Anthropomorphism
```suggestion
After you add the `OperatorPolicy` policy template, the operatorGroup` and `subscription` objects are created on the cluster by using the controller. This prompts the OLM to complete the rest of the installation process. You can view the health of owned resources in the `.status.Conditions` and `.status.relatedObjects` fields of the `OperatorPolicy` resource on the managed cluster.
```
